CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -1726,43 +1810,51 @@ public let DECL_NODES: [Node] = [
         name: "Attributes",
         kind: .collection(kind: "AttributeList", collectionElementName: "Attribute"),
         nameForDiagnostics: "attributes",
+        description: "Attributes attached to the protocol declaration.",
```

Comment:
Could you add examples of attributes and modifiers here as well? I think they could be helpful to understand the difference between the two.